tcks/microprofile-rest-client/tck-suite.xml

```diff
@@ -17,44 +17,48 @@
     <test name="microprofile-rest-client 1.2 TCK">
         <packages>
             <package
-                name="org.eclipse.microprofile.rest.client.tck.*">
+                    name="org.eclipse.microprofile.rest.client.tck.*">
                 <!-- timeout not implemented -->
                 <exclude name="org.eclipse.microprofile.rest.client.tck.timeout"/>
             </package>
         </packages>
-        
+
         <classes>
-            <!-- mp/scope not implemented -->
-            <class name="org.eclipse.microprofile.rest.client.tck.cditests.HasAppScopeTest">
+            <!-- mp/scope not supported for ConversationScope and SessionScope-->
+            <class name="org.eclipse.microprofile.rest.client.tck.cditests.HasConversationScopeTest">
                 <methods>
                     <exclude name=".*" />
                 </methods>
             </class>
-            <class name="org.eclipse.microprofile.rest.client.tck.cditests.HasConversationScopeTest">
+            <class name="org.eclipse.microprofile.rest.client.tck.cditests.HasSessionScopeTest">
                 <methods>
                     <exclude name=".*" />
                 </methods>
             </class>
-            <class name="org.eclipse.microprofile.rest.client.tck.cditests.HasRequestScopeTest">
+
+
+            <!-- excluded because of race scope configuration on SimpleGetApi interface -->
```

Comment:
@mkouba I added a comment about here. If you run the tests one by one, they all pass. If you run them together, they fail randomly: one pass and not the other, etc.

Reply:
That's really weird and we should probably try to find the real cause first. I'll try to run the tests locally.

Reply:
Thanks.

Reply:
So I think that the problem is that `ConfigProvider.getConfig()` caches the `ConfigProviderResolver` in a static field. We do work around this limitation in [QuarkusConfigProducer](https://github.com/quarkusio/quarkus/blob/master/extensions/arc/runtime/src/main/java/io/quarkus/arc/runtime/QuarkusConfigProducer.java#L29) by using `ConfigProviderResolver.instance().getConfig()` instead. I'll try to update the code and run the tests.

Reply:
Hm, it didn't help most probably because the rest client beans are registered in STATIC_INIT and `ConfigProviderResolver` is re-set during RUNTIME_INIT...

Reply:
Hey @mkouba, thanks for the investigation. How do you advise we proceed regarding this?

Reply:
I've created https://github.com/quarkusio/quarkus/issues/3089 - pls add this link to the relevant comment in the tck-suite.xml.

Then I think we can merge this PR.

Reply:
Done